lms/envs/test.py

```diff
@@ -484,6 +484,9 @@
 # Enable teams feature for tests.
 FEATURES['ENABLE_TEAMS'] = True
 
+# Enable indexing teams for search
+FEATURES['ENABLE_TEAMS_INDEX'] = True
```

Comment:
You should put this flag also in common.py with a value of False. I imagine we will want devops to enable it via configuration on edx (instead of making the value in common.py True, check with @maxrothman) since it requires also standing up edx-search, and open source users may not have that.